CONTRIBUTE.md

```diff
@@ -5,7 +5,7 @@ before reporting things, and help us with the open ones.
 
 Guidelines:
 
-- read the [libp2p spec](https://github.com/ipfs/specs/tree/master/protocol/network)
+- read the [libp2p spec](https://github.com/ipfs/specs/tree/master/libp2p)
```

Comment:
This actually lives in github.com/libp2p/specs now (cc @RichardLitt the old ones need to be removed)